nibabel/xmlutils.py

```diff
@@ -27,7 +27,11 @@ def _to_xml_element(self):
     def to_xml(self, enc='utf-8'):
         """ Output should be an xml string with the given encoding.
         (default: utf-8)"""
-        return tostring(self._to_xml_element(), enc)
+        ele = self._to_xml_element()
+        if ele is None:
```

Comment:
I prefer @matthew-brett 's syntax. I see those phrases a lot, so they read easy for me.

But I prefer `return ele or ''` for readability above both... if `ele` can't be `False`/`[]`/etc.